open-company/open-company-web
# Private board invite

Card: https://trello.com/c/RnKiS5Fl

To test:
- using a user that is logged in with slack
- create a private board
- check that you can add and remove users and change permission of the private board
- check that you can't add slack user that didn't join the carrot team yet
- now change the board permissions to public
- refresh the window
- open board settings again
- do you NOT see a UI jump in the view? Good
- change board permissions back to private
- do you still see the list of users you setup before? Good
- make any other test you can think of with a private board
